example/ios/Podfile.lock

```diff
@@ -77,7 +77,7 @@ PODS:
   - libevent (2.1.12)
   - NWWebSocket (0.5.2)
   - OpenSSL-Universal (1.1.1100)
-  - pusher-websocket-react-native (1.1.0):
+  - pusher-websocket-react-native (1.1.1):
```

Comment:
Shouldn't it be updated to the latest version? `1.2.0`

Reply:
No, because it's not released yet :)